pkg/operator/etcd_assets/bindata.go

```diff
@@ -808,6 +808,25 @@ ${COMPUTED_ENV_VARS}
       failureThreshold: 3
       periodSeconds: 5
       successThreshold: 1
+    livenessProbe:
+      exec:
+        command:
+          - /bin/bash
+          - -c
+          - |
+            set -xe
+            # liveness probe uses a serialized get to an non-existant key (liveness) as a condition.
+            unset ETCDCTL_ENDPOINTS
+            /usr/bin/etcdctl \
```

Comment:
I would measure an e2e run from this PR and compare with another PR. I think you can load the prometheus snapshots and compare the cpu numbers from there.

Reply:
I have never done this, can we pair on this ? .. I would appreciate if you show me how :)

Reply:
I'm pretty sure you've done this before with using the prow url to load into https://promecieus.dptools.openshift.org/?search= - but we can pair on this, sure.

Reply:
I actually think that this might be a better approach:
https://github.com/openshift/cluster-etcd-operator/pull/478/files

this seems to also check leadership and other alarms and is easier on the TLS handshakes:
https://github.com/etcd-io/etcd/blob/f09da32f9df217d603884475e195bfe4f81550ae/server/etcdserver/api/etcdhttp/health.go#L47-L57

can we use this instead?